AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/RemoteParticipantUpdate.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.samples.communication.calling.external.calling;
+
+import com.azure.android.communication.calling.RemoteParticipant;
+
+public class RemoteParticipantUpdate {
+
+    private final RemoteParticipant remoteParticipant;
```

Comment:
can we save ID here instead of RemoteParticipant? then no need to call below line in receiver callingContext.getId(remoteParticipantUpdate.getRemoteParticipant())

Reply:
Discussed offline - decided to leave as is